distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -136,6 +136,17 @@ public static void deleteMetacard(String id) {
                 .all();
     }
 
+    public static void deleteMetacard(String id, boolean checkResponse) {
+        if (checkResponse) {
+            deleteMetacard(id);
+        } else {
+            LOGGER.info("Deleting metacard {}", id);
```

Comment:
Do you really want to log every time (at info level) you are attempting to delete a metacard in a test?